NORTRIP_inputparams_subroutines.f90

```diff
@@ -613,6 +613,7 @@ subroutine read_NORTRIP_flags
     use_traffic_turb_flag=match_string_int('use_traffic_turb_flag',unit_in,unit_logfile_temp,0)
     use_melt_freeze_energy_flag=match_string_int('use_melt_freeze_energy_flag',unit_in,unit_logfile_temp,0)
     use_stability_flag=match_string_int('use_stability_flag',unit_in,unit_logfile_temp,use_stability_flag)
+    use_energy_correction_flag=match_string_int('use_energy_correction_flag',unit_in,unit_logfile_temp,use_energy_correction_flag)
```

Comment:
Should add a default to this function (0 or 1). However, I get a "no match found" error when trying, and I don't understand why.